CONTRIBUTING.md

```diff
@@ -335,11 +335,18 @@ Mocha tests are contained in `__tests__` directories.
 #### Jest
 Jest tests are stored in the same directory as source code files with the `.test.js` suffix.
 
-### Running Jest Unit Tests
-
-```bash
-node scripts/jest
-```
+### Running Kibana tests
+
+The following table outlines possible test file locations and how to invoke them:
+
+| Test location                                                                                                                                           | Runner command (working directory is kibana root)                                       |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| `src/**/*.test.js`<br>`src/**/*.test.ts`                                                                                                                | `yarn test:jest -t=regexp [test path]`                                                  |
```

Comment:
`-t` matches on test name, not path.
I use `--testPathPattern`. it MUCH faster, as jest runs only on specified file

Reply:
I've tried to make the docs clearer and added some examples to illustrate how I think people might use this.